concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1642,38 +1650,40 @@ jobs:
       - get: cf-secrets
         passed: ['cf-deploy']
 
-      - task: create-temp-user
-        file: paas-cf/concourse/tasks/create_admin.yml
-        config:
-          params:
-            PREFIX: continuous-smoketest-user
-
-      - task: smoke-tests-config
-        file: paas-cf/concourse/tasks/smoke-tests-config.yml
-
-      - task: smoke-tests-run
-        file: paas-cf/concourse/tasks/smoke-tests-run.yml
-        on_failure:
-          task: alert
+      - do:
+        - task: create-temp-user
+          file: paas-cf/concourse/tasks/create_admin.yml
           config:
-            platform: linux
-            image: docker:///governmentpaas/awscli
             params:
-              AWS_DEFAULT_REGION: {{aws_region}}
-              DEPLOY_ENV: {{deploy_env}}
-              SYSTEM_DNS_ZONE_NAME: {{system_dns_zone_name}}
-              ALERT_EMAIL_ADDRESS: {{ALERT_EMAIL_ADDRESS}}
-            inputs:
-              - name: paas-cf
-              - name: smoke-tests-log
-            run:
-              path: sh
-              args:
-              - -e
-              - -c
-              - |
-                paas-cf/concourse/scripts/smoke_tests_email.sh \
-                  ${DEPLOY_ENV} ${SYSTEM_DNS_ZONE_NAME} ${ALERT_EMAIL_ADDRESS}
+              PREFIX: continuous-smoketest-user
```

Comment:
As commented in person, this won't work in the newer versions of concourse (>=1.1) because it complains:

```
DEPRECATION WARNING: Specifying both `file` and `config.params` in a task step is deprecated, use params on task step directly
``` 

I think we will have to remove the `config` key. 

But that is a issue that should be fix in the story about upgrading concourse.